polygon-digital-carbon/src/templates/CarbonmarkCreditToken.ts

```diff
@@ -0,0 +1,48 @@
+import { Transfer } from '../../generated/BCT/ERC20'
+import { handleCreditTransfer } from '../TransferHandler'
+import { Retired } from '../../generated/CarbonmarkCreditTokenFactory/CarbonmarkCreditToken'
+import { ZERO_BI } from '../../../lib/utils/Decimals'
+import { loadCarbonCredit } from '../utils/CarbonCredit'
+import { incrementAccountRetirements, loadOrCreateAccount } from '../utils/Account'
+import { saveRetire } from '../utils/Retire'
+import { ZERO_ADDRESS } from '../../../lib/utils/Constants'
+
+
+export function handleCarbonmarkCreditTransfer(event: Transfer): void {

```

Comment:
you probably don't need to handle this with a separate handler. handleCreditTransfer will already catch the Transfer event from this data source, so putting that in the manifest with the correct file path should catch the event.

Reply:
agreed, keeping handlers consolidated simplifies the code structure and eases long term maintenance

Reply:
Done in acb0918
My rationale for that was to have all handlers for a particular contract in the same file. Even if they were just passthough for other handlers (kind of using a lib). As a noob I had a hard time figuring out how the subgraph.yml file connected with the files contaning the handlers.